src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -1279,6 +1279,31 @@ public void T036_CreateRelationshipWithId()
             packagePath1.Delete();
         }
 
+        [Fact]
+        public void OpenInternalTargetRelationships()
+        {
+            // This is to test different behavior on Mono vs .NET Core
```

Comment:
Is this test expected to pass on one but succeed on the other?

Reply:
This should pass on both. I wasn't sure the value of the test since it doesn't test the change really on .NET Core, but if Mono shares the tests, then that's awesome.